.github/workflows/tests.yml

```diff
@@ -0,0 +1,68 @@
+name: "Tests"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        node_version: ['lts/*', 'node']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Install Node
+        uses: dcodeIO/setup-node-nvm@master
```

Comment:
I think I would rather use the official node.js action and run tests on mac and windows as well as linux.

Reply:
> Is this only used to reference lts?

Yes, there is no way to reference `lts` or `latest` with the official node.js action. And updating the version number will be forgotten if we don't automate it.

Reply:
> and run tests on mac and window

I don't really see how this library can change depending on which os it is on. It has no dependencies and no os dependent code.

Reply:
I suppose maybe testing the cli but we don't do that right now anyway.